packages/cli/src/main/kotlin/elide/tool/io/RuntimeWorkdirManager.kt

```diff
@@ -92,12 +95,17 @@ import elide.tool.io.WorkdirManager.WorkdirHandle
   @Volatile private lateinit var tempDirectory: Path
   private val handleCache: SortedMap<File, WorkdirHandle> = ConcurrentSkipListMap()
 
-  private fun currentSharedTempPrefix(): String {
-    return StringBuilder().apply {
-      append(runtimeDirPrefix)
-      append(Elide.version())
-    }.toString()
-  }
+  // Directory providers from dirs.dev library.
+  private val baseDirectories: BaseDirectories = BaseDirectories.get()
+  private val userDirectories: UserDirectories = UserDirectories.get()
+  private val projectDirectories: ProjectDirectories = ProjectDirectories.from("dev", "elide", "elide")
+
+  // private fun currentSharedTempPrefix(): String {
+    // return StringBuilder().apply {
+      // append(runtimeDirPrefix)
+      // append(Elide.version())
+    // }.toString()
+  // }
```

Comment:
this will get easier to review, and easier to work with, if you remove code instead of commenting. i get the impulse